src/distributions/other.rs

```diff
@@ -44,15 +44,21 @@ pub struct Alphanumeric;
 impl Distribution<char> for Standard {
     #[inline]
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> char {
-        let range = Uniform::new(0u32, 0x11_0000);
```

Comment:
Could we make this `char::MAX as u32`? More explicit about what we're doing

Reply:
This line is being removed. But I don't think this is a good idea; you're off by 1 (should be inclusive) and if `char::MAX` were ever to change we don't know now whether we should use the whole range (minus the existing gap). So better just to use local constants as in the current implementation.